build.gradle

```diff
@@ -339,6 +339,13 @@ subprojects {
     addParametersForTests(name, options)
   }
 
+  java {
+    // We should only set this if Java version is < 9 (--release is recommended for >= 9), but the Scala plugin for IntelliJ sets
+    // `-target` incorrectly if this is unset
```

Comment:
Why can't we set --release instead of source/target?

Reply:
We are setting -release flag for scala compile https://github.com/apache/kafka/blob/trunk/build.gradle#L771
But as mentioned in the code comment, some how its not working without above properties

Reply:
I found that setting `ScalaCompile.sourceCompatibility` ,`ScalaCompile.targetCompatibility` is fixed the issue. so i have updated the same.